src/GraphQL.Analyzers.Tests/InputGraphTypeAnalyzerTests.Constructors.cs

```diff
@@ -31,6 +34,18 @@ public CustomInputObjectGraphType()
                   }
               }
 
+              public class AutoCustomInputObjectGraphType : AutoRegisteringInputObjectGraphType<{|#1:MySource|}>
+              {
+              }
+
+              public class MyGraphType : ObjectGraphType
+              {
+                  public MyGraphType()
+                  {
+                      Field<AutoRegisteringInputObjectGraphType<{|#2:MySource|}>>();
+                  }
+              }
+
```

Comment:
I extended one test to include a type derived from `AutoRegisteringInputObjectGraphType` and usage of `AutoRegisteringInputObjectGraphType`. The code to analyze these cases is the same as before. The only difference is the diagnostic location.